Client/Frontend/Browser/TabManager.swift

```diff
@@ -766,9 +766,14 @@ class TabManager: NSObject {
         imageStore?.clearExcluding(savedUUIDs)
     }
 
-    fileprivate lazy var restoreTabsInternal: () -> Tab? = {
+    fileprivate static var restoreOnceToken = false
```

Comment:
I think using `dispatch_once_t` is probably going to be better. It was designed for things like this, as opposed to a flat boolean.

Reply:
`dispatch_once_t` is no longer available in swift. The guide now promotes this method or lazy var for such purposes